UPGRADE-6.8.md

```diff
@@ -159,35 +262,51 @@ foreach ($entities as $entity) {
+Profiles are now identified and displayed only by their technical name.
+- The `$label` property and the following methods in `Shopware\Core\Content\ImportExport\ImportExportProfileEntity` have been removed:
+  - `getLabel()`
+  - `setLabel()`
+  - `getTranslations()`
+  - `setTranslations()`
+- The following classes have been removed:
+  - `Shopware\Core\Content\ImportExport\ImportExportProfileTranslationCollection`
+  - `Shopware\Core\Content\ImportExport\ImportExportProfileTranslationDefinition`
+  - `Shopware\Core\Content\ImportExport\ImportExportProfileTranslationEntity`
+- `createLog()` and `getConfig()` in `Shopware\Core\Content\ImportExport\Service\ImportExportService` now use `$technicalName` instead of `$label` when generating filenames.
+- `generateFilename()` in `Shopware\Core\Content\ImportExport\Service\FileService` now uses `$technicalName` instead of `$label` as profile name.
 
-For user interfaces that display only one delivery & transaction, there is now a new reference in the order for a `primaryOrderDelivery` or `primaryOrderTransaction`.
-If an extension modifies or adds new deliveries or transactions, this should be taken into account.
-To partly comply with old behaviour, primary deliveries are ordered first and primary transactions are ordered last wherever appropriate.
+## ApiClient confidential flag
 
-* Replace delivery accesses like `order.deliveries.first()` or `order.deliveries[0]` with `order.primaryOrderDelivery`
-* Replace transaction accesses like `order.transactions.last()` or `order.transactions[length - 1]` with `order.primaryOrderDelivery`
+* You must explicitly pass a boolean value to the `confidential` parameter  of `\Shopware\Core\Framework\Api\OAuth\Client\ApiClient`.
+* You must pass the `confidential` parameter as the third parameter of the constructor.
+* You must pass the `name` parameter as the fourth parameter of the constructor.
 
-## Payment: Removal of Payment Method "Debit Payment"
+</details>
 
-The payment method `DebitPayment` has been removed as it did not fulfill its purpose.
-If the payment method is and was not used, it will be removed.
-Otherwise, the payment method will be disabled.
+# Administration
 
-## Remove route `widgets.account.order.detail`:
+<details>
 
-* Remove all references to `widgets.account.order.detail` and ensure that affected components handle navigation and display correctly
+## Removal of "sw-empty-state"
+* The old `sw-empty-state` component will be removed in the next major version. Please use the new `mt-empty-state` component instead.
+
+Before:
+```html
+<sw-empty-state title="short title" subline="longer subline" />
+```
+After:
+```html
+<mt-empty-state title="short title" description="longer description"/>
+```
 
 ## Removal of $tc function:
 
 * The `$tc` function will be completely removed
 * All translation calls should use `$t` instead
 
-
-## Introduced in 6.7.0.0
-
 ## Settings Menu Structure was changed
```

Comment:
This change is already in place and was a break for 6.7.0.0 . I don't know why it's still in here, the mentioned twig blocks are already gone 😬